application/controllers/ThemeOptionsController.php

```diff
@@ -166,14 +170,16 @@ public function actionResetMultiple()
      */
     public function actionUninstallMultiple()
     {
-        $aTemplates = json_decode(App()->request->getPost('sItems'));
-        $gridid = App()->request->getPost('grididvalue'); //what is gridid ???
+        $aTemplates = json_decode(App()->request->getPost('sItems')); //array of ids
+
+        //can be 'themeoptions-grid' (for survey themes) or 'questionthemes-grid'
+        $gridid = App()->request->getPost('grididvalue');
         $aResults = array();
 
         if (Permission::model()->hasGlobalPermission('templates', 'update')) {
             foreach ($aTemplates as $template) {
                 $templateID = (int) $template;
-                $model = $this->loadModel($templateID, $gridid);
+                $model = $this->loadModel($templateID, $gridid); //model is TemplateConfiguration or QuestionTheme
```

Comment:
Maybe better 
```
if(!$templateID) {
continue;
}
```

Don't try to search TemplateConfiguration::model()->findByPk(0)